scripts/build-js-loader.js

```diff
@@ -0,0 +1,43 @@
+const {minify} = require('terser');
+const fs = require('fs');
+
+const header = `{% load sentry_helpers %}`;
+const loaderScriptPath = './src/sentry/templates/sentry/js-sdk-loader.js';
+const loaderTmplPath = './src/sentry/templates/sentry/js-sdk-loader.js.tmpl';
+const loaderMinTmplPath = './src/sentry/templates/sentry/js-sdk-loader.min.js.tmpl';
+
+async function run() {
+  const base = fs.readFileSync(loaderScriptPath, 'utf-8');
+
+  const unminifiedLoader = `${header}${replacePlaceholders(base)}`;
+  const {code: minifiedBase} = await minify(base, {
+    toplevel: false,
+    ecma: 5,
+    mangle: {
+      reserved: ['onLoad', 'forceLoad'],
+    },
+    format: {
+      ecma: 5,
+    },
+  });
+
+  const minifiedLoader = `${header}${replacePlaceholders(minifiedBase)}\n`;
+
+  fs.writeFileSync(loaderTmplPath, unminifiedLoader, 'utf-8');
+
+  console.log(`Updated loader script template at ${loaderTmplPath}`);
+
+  fs.writeFileSync(loaderMinTmplPath, minifiedLoader, 'utf-8');
+
+  console.log(`Updated loader min. script template at ${loaderMinTmplPath}`);
+}
+
+function replacePlaceholders(str) {
+  return str
+    .replace('__LOADER__PUBLIC_KEY__', "'{{ publicKey|safe }}'")
+    .replace('__LOADER_SDK_URL__', "'{{ jsSdkUrl|safe }}'")
+    .replace('__LOADER__CONFIG__', '{{ config|to_json|safe }}')
+    .replace('__LOADER__IS_LAZY__', '{{ isLazy|safe|lower }}');
+}
```

Comment:
Might be nice to have some comments on what is going on here

Reply:
I added a comment there!